governance/program/src/state/realm.rs

```diff
@@ -8,11 +8,32 @@ use solana_program::{
 
 use crate::{
     error::GovernanceError,
+    state::enums::{GovernanceAccountType, MintMaxVoteWeightSource},
     tools::account::{assert_is_valid_account, get_account_data, AccountMaxSize},
     PROGRAM_AUTHORITY_SEED,
 };
 
-use crate::state::enums::GovernanceAccountType;
+/// Realm Config defining Realm parameters.
+#[repr(C)]
+#[derive(Clone, Debug, PartialEq, BorshDeserialize, BorshSerialize, BorshSchema)]
+pub struct RealmConfig {
+    /// Optional council mint
+    pub council_mint: Option<Pubkey>,
+
+    /// The source used for community mint max vote weight source
+    /// Note: This field is not used yet. It's reserved for future versions
+    pub community_mint_max_vote_weight_source: MintMaxVoteWeightSource,
+
+    /// An authority tasked with none critical and maintenance Realm operations
+    /// For example custodian authority is required to add governances to the Realm
+    /// There is no security risk with adding governances to the Realm but it should not be open for everybody
+    /// to prevent unrelated entries and noise
+    /// Note: This field is not used yet. It's reserved for future versions
+    pub custodian: Option<Pubkey>,
+
+    /// Reserved space for future versions
+    pub reserved: [u8; 8],
```

Comment:
Since you have the reserved bytes before the `Option` and `String` earlier, maybe it makes sense to do the same thing here, and put `council_mint` later in the struct as well

Reply:
Done